windows/provision-jenkins-slave.ps1

```diff
@@ -41,11 +41,8 @@ Install-ChocolateyShortcut `
 
 
 # install the JRE.
-choco install -y server-jre8
+choco install -y openjdk
 Update-SessionEnvironment
-Write-Output 'Enabling the unlimited JCE policy...'
```

Comment:
@shanejonas any comment on this? 

Im assuming this is superseded by openjdk?

Reply:
OpenJDK has unlimited policy as a default: https://github.com/open-eid/cdoc4j/wiki/Enabling-Unlimited-Strength-Jurisdiction-Policy